tests/QAN/pages/qanFiltersFragment.js

```diff
@@ -150,6 +150,15 @@ module.exports = {
     I.forceClick(showAllLink);
   },
 
+  async applyShowAllLinkIfItIsVisible(groupName) {
+    const showAllLink = this.getFilterGroupCountSelector(groupName);
+    const numOfShowAllLinkSectionCount = await I.grabNumberOfVisibleElements(showAllLink);
+
+    if (numOfShowAllLinkSectionCount === 1) {
```

Comment:
it can be just `if (numOfShowAllLinkSectionCount) {`

Reply:
ok.thanks